### ISSUE TITLE
User documentation

### ISSUE THREAD

**Issue by mrocklin** [libdynd/dynd-python#738]
It would be nice to have user facing documentation, even if it's just API docs.

I use and recommend readthedocs/sphinx.  It's very easy to get set up.

**Comment by insertinterestingnamehere** [comment]
Right. I've spent some time working on integrating Doxygen into Sphinx using Breathe. It's sort-of working now, but still needs some work. More on this soon.
